colossalai/shardformer/README.md

```diff
@@ -390,3 +391,45 @@ To validate that training the model using shardformers does not impact its conve
 | 0.81855  | 0.87124 | 0.10357 |     1      |    False    |
 
 Overall, the results demonstrate that using shardformers during model training does not affect the convergence.
+
+### Performance
+
+
+We also conducted [benchmark tests](./examples/performance_benchmark.py) to evaluate the performance improvement of Shardformer. We compared the training time between the original model and the shard model.
```

Comment:
This is a duplicated section header. This should be put at the `System Performance` section.